docs/style.css

```diff
@@ -36,12 +50,33 @@
   color: var(--theme-foreground-faint);
 }
 
-.observablehq-link-active a[target=_blank]:not(:hover, :focus)::after {
+.observablehq-link-active a[target="_blank"]:not(:hover, :focus)::after {
   color: var(--theme-foreground-focus);
 }
 
 #observablehq-main {
   min-height: calc(100vh - 23.8rem);
+  font-size: 16px;
+  font-weight: 500;
+}
+
+p a[href] {
+  color: inherit;
+  text-decoration: underline;
+}
+
+h1 {
+  font-family: "Spline Sans Mono", var(--monospace);
+  font-weight: 500;
+  margin-bottom: 1rem;
```

Comment:
What are these global margin changes to heading elements for? This will require extensively checking all the pages to make sure it works as intended.

Reply:
I'm deferring to @CobusT on this one.

Reply:
They are not needed here. I removed them.